wagtail/documents/views/bulk_actions/document_bulk_action.py

```diff
@@ -0,0 +1,19 @@
+from wagtail.admin.views.bulk_action import BulkAction
+from wagtail.documents import get_document_model
+from wagtail.documents.permissions import permission_policy as documents_permission_policy
+
+
+class DocumentBulkAction(BulkAction):
```

Comment:
this and `PageBulkAction` don't seem to implement `display_name`, `action_type` nor `aria_label`.  feels like some tidy up is needed

Reply:
These aren't meant to be used as bulk actions, rather templates for the specific actions. Trying to create an object from either of these classes will raise an Exception